packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -118,19 +125,34 @@ class MouseTrackerAnnotation {
   ///  * [onEnter], which is triggered when a mouse pointer enters the region.
   final PointerExitEventListener onExit;
 
+  /// Controls how one annotation replaces another annotation during an
+  /// annotation search.
+  ///
+  /// During the process of determining pointer entrance and exit, two
+  /// annotations are considered equal if the two annotations have [key]s that
+  /// are equal by [operator==].
+  ///
+  /// The [key] must not be null.
+  ///
+  /// See also:
+  ///
+  ///  * [UniqueKey], [ObjectKey], and [ValueKey], which are common classes of
+  ///    keys.
+  final Key key;
```

Comment:
that should probably be LocalKey, i don't think we want to allow a GlobalKey here